tools/cert-sidecar/example/docker-compose.yml

```diff
@@ -0,0 +1,164 @@
+version: '3.8'
+services:
+
+  # automatically restarting unhealty containers
```

Comment:
It is probably not the best strategy, but it is just for an example. We have to discuss better.

Reply:
Services using node obtain certificate files only when they are started